apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-queue-management-manualselection.spec.ts

```diff
@@ -239,3 +239,159 @@ test.describe('OC - Contact Manager Routing', () => {
 		});
 	});
 });
+
```

Comment:
This test file is getting pretty big. It might be a good idea to create a new file for these new test cases. Smaller files are easier to read and debug. Wdyt?